### PR TITLE
Added support for keccak-224/384/512

### PR DESCRIPTION
**Pull request by tabrath**

### CODE REVIEW

File: sum.go [hunk 1]
```diff
@@ -7,10 +7,10 @@ import (
 	"errors"
 	"fmt"
 
-	keccak "github.com/ethereum/go-ethereum/crypto/sha3"
 	blake2b "golang.org/x/crypto/blake2b"
 	blake2s "golang.org/x/crypto/blake2s"
 	sha3 "golang.org/x/crypto/sha3"
+	keccak "leb.io/hashland/keccakpg"
```
Comment (whyrusleeping) [inline sum.go]: I'll need to vet this lib vs the ethereum one. Also it will need to be vendored with gx

Comment (tabrath) [inline sum.go]: Ok, anything I can do?

Comment (whyrusleeping) [inline sum.go]: Nope, this looks good. I'll just need to gx vendor the lib

**Comment by whyrusleeping** [comment]
merged in #49, thanks!

**Comment by tabrath** [comment]
Happy to contribute!

**Pull request closed without merging**